.gitignore

```diff
@@ -7,19 +7,14 @@ target
 
 # BlueJ files
 *.ctxt
-.gradle
```

Comment:
this dir should be ignored. Please rollback the row

Reply:
✅